share/setup.nsi.in

```diff
@@ -77,6 +77,8 @@ Section -Main SEC0000
     File /oname=readme.txt @abs_top_srcdir@/doc/README_windows.txt
     SetOutPath $INSTDIR\share\rpcauth
     File /r /x @abs_top_srcdir@/share\rpcauth\*.*
+    SetOutPath $INSTDIR\share\examples
+    File @abs_top_srcdir@/share\examples\bitcoin.conf
```

Comment:
should this also be included in the uninstall section below? L131-L136

Reply:
It should be covered by removing the share directory?

Reply:
:doh: you're right, i missed that